src/components/NurseryWithdrawals/PlantingProgressList.tsx

```diff
@@ -1,7 +1,116 @@
 import { useAppSelector } from 'src/redux/store';
 import { searchPlantingProgress } from 'src/redux/features/plantings/plantingsSelectors';
+import CellRenderer, { TableRowType } from 'src/components/common/table/TableCellRenderer';
+import { RendererProps } from 'src/components/common/table/types';
+import Table from 'src/components/common/table';
+import Link from 'src/components/common/Link';
 
-export default function PlantingProgressList(): JSX.Element {
-  const data = useAppSelector((state: any) => searchPlantingProgress(state, ''));
-  return <p>I am a list {data.length}</p>;
+const useStyles = makeStyles(() => ({
+  text: {
+    fontSize: '14px',
+    '& > p': {
+      fontSize: '14px',
+    },
+  },
+}));
+
+const columnsWithoutZones = (): TableColumnType[] => [
+  {
+    key: 'siteName',
+    name: strings.PLANTING_SITE,
+    type: 'string',
+  },
+  {
+    key: 'totalSeedlingsSent',
+    name: strings.TOTAL_SEEDLINGS_SENT,
+    type: 'number',
+  },
+];
+
+const columnsWithZones = (): TableColumnType[] => [
+  {
+    key: 'subzoneName',
+    name: strings.SUBZONE,
+    type: 'string',
+  },
+  {
+    key: 'plantingCompleted',
+    name: strings.PLANTING_COMPLETE,
+    tooltipTitle: strings.PLANTING_COMPLETE_TOOLTIP,
+    type: 'boolean',
+  },
+  {
+    key: 'siteName',
+    name: strings.PLANTING_SITE,
+    type: 'string',
+  },
+  {
+    key: 'zoneName',
+    name: strings.ZONE,
+    type: 'string',
+  },
+  {
+    key: 'targetPlantingDensity',
+    name: strings.TARGET_PLANTING_DENSITY,
+    tooltipTitle: strings.TARGET_PLANTING_DENSITY_TOOLTIP,
+    type: 'number',
+  },
+  {
+    key: 'totalSeedlingsSent',
+    name: strings.TOTAL_SEEDLINGS_SENT,
+    type: 'number',
+  },
+];
+
+export type PlantingProgressListProps = {
+  search: string;
+  plantingCompleted?: boolean;
+};
+
+export default function PlantingProgressList({ search, plantingCompleted }: PlantingProgressListProps): JSX.Element {
+  const classes = useStyles();
+  const data = useAppSelector((state: any) => searchPlantingProgress(state, search, plantingCompleted));
+
+  const hasZones = data?.some((d) => d.subzoneName);
+
+  if (!data) {
+    return <CircularProgress sx={{ margin: 'auto' }} />;
+  }
+
+  return (
+    <Box>
+      <Table
+        id='plantings-progress-table'
+        columns={hasZones ? columnsWithZones : columnsWithoutZones}
+        rows={data}
+        orderBy={hasZones ? 'subzoneName' : 'siteName'}
+        Renderer={DetailsRenderer(classes)}
+      />
+    </Box>
+  );
 }
+
+const DetailsRenderer =
+  (classes: any) =>
+  (props: RendererProps<TableRowType>): JSX.Element => {
+    const { column, row } = props;
+
+    const createLinkToWithdrawals = () => {
+      const filterParam = row.subzoneName
+        ? `subzoneName=${encodeURIComponent(row.subzoneName)}`
+        : `siteName=${encodeURIComponent(row.siteName)}`;
+      const url = `${APP_PATHS.NURSERY_WITHDRAWALS}?tab=${strings.WITHDRAWAL_HISTORY}&${filterParam}`;
```

Comment:
the filter param is not used here, will need a follow up PR in withdrawals to filter by param and also clear the param if not relevant